### PR TITLE
MFW-3167-Update-Schema

### PR DESCRIPTION
**Pull request by AlistairJohnParker**

### CODE REVIEW

**Review by rasmussen-untangle — APPROVED** [review]

File: v1/policy_manager/test_settings.json [hunk 1]
```diff
@@ -211,18 +211,15 @@
                 "type": "ServiceEndpoint",
                 "items": [
                     {
-                        "protocol": "TCP",
-                        "ipspecifier": "12.34.56.78",
+                        "protocol": "6",
```
Comment (jsommerville-untangle) [inline v1/policy_manager/test_settings.json:214]: should be int

Comment (AlistairJohnParker) [reply v1/policy_manager/test_settings.json:214]: Agreed

File: validate.py [hunk 2]
```diff
@@ -5,6 +5,190 @@
 import jsonschema
 import sys
 
+def printCondition(prefix,c):
```
Comment (jsommerville-untangle) [inline validate.py:8]: add comments for these functions

File: validate.py [hunk 3]
```diff
@@ -5,6 +5,190 @@
...
+groups = {}
+
+def printGroup(prefix, g):
+    print(prefix,'Group:', g['id'], '\tName:',g['name'], '\tItems:',g['items'])
+
+def checkGroupItems(prefix, g, items):
+    errors = 0
+    type = g['type']
+    if items == None or len(items) == 0:
+        print('\t'+prefix, "Error - Group wtih empty items list")
```
Comment (jsommerville-untangle) [inline validate.py:74]: typo

**Pull request merged**